CHANGELOG.md

```diff
@@ -805,7 +806,7 @@ Initial release.
   [MariusV]: https://github.com/MariusVolkhart
   [SaketN]: https://github.com/saket
   [RomanZ]: https://github.com/romtsn
-  [ZacS]: https://github.com/ZacSweers
+  [ZacSweers]: https://github.com/ZacSweers
```

Comment:
I don't know if the changelog and this list are autogenerated through.

Reply:
I don't know either, but I thought I'd help Zac:

<img width="835" alt="Screen Shot 2022-10-04 at 12 05 38" src="https://user-images.githubusercontent.com/5759366/193792809-1e9eab04-1d3a-48a8-9349-9547670db99f.png">

Reply:
not autogenerated